cmd/terramate/cli/cloud.go

```diff
@@ -367,6 +368,10 @@ func (c *cli) setupCloudConfig(requestedFeatures []string) error {
 		c.cloud.run.orgUUID = activeOrgs[0].UUID
 	}
 
+	if len(c.cloud.run.orgUUID) > 0 {
```

Comment:
```suggestion
	if c.cloud.run.orgUUID != "" {
```
wdyt?

it looks like it's a slice using len() :)